libs/protocol-ui/src/components/molecules/Select/Select.tsx

```diff
@@ -45,8 +53,9 @@ export interface SelectProps {
   isDisabled?: boolean;
   variant?: 'outline' | 'filled';
   value?: any;
-
+  innerProps?: any;
```

Comment:
Can we be more specific with this type

Reply:
@alexkeating @jonathanprozzi this might have to be a debt.